modules/animation/include/modules/animation/datastructures/keyframesequence.h

```diff
@@ -74,6 +74,14 @@ class IVW_MODULE_ANIMATION_API KeyframeSequence : public Serializable,
 
     Seconds getFirstTime() const;
     Seconds getLastTime() const;
+    /*
+     * Return time of previous keyframe, or at if not found.
+     */
+    virtual Seconds getPrevTime(Seconds at) const = 0;
```

Comment:
would make more sense to return an std::optional<Seconds> here

Reply:
Definitely, fixed